docs/journal/01-cold-storage.md

```diff
@@ -76,14 +92,13 @@ In this configuration, devices post event data to a Service Bus topic. Worker ro
 
 - **Throughput.** A worker role has less overhead than a web role (*NEED SOMEONE TO VERIFY THIS, AND MORE DETAIL ON WHY*)
 
-- **Reliability.** The AMQP protocol supports reliability guarantees, helping to ensure that event data is not lost when sent to a topic. Additionally, a subscription can be transactional; if a worker role fails while processing the data for an event this data is not lost but is instead returned to the Service Bus topic from whence it can be retrieved again.
+- **Reliability.** The AMQP protocol supports reliability guarantees, helping to ensure that event data is not lost when sent to a topic. Additionally, a subscription can be transactional; if a worker role fails while processing an event, the event is returned to the Service Bus topic and can be retrieved again.
```

Comment:
Does the use of AMQP really make a difference? I mean, if the HTTP doesn't get back a 200 (or whatevs) then the sender knows that the event wasn't received. I think that the important bit here is the second point: 
> if a worker role fails while processing an event, the event is returned to the Service Bus topic and can be retrieved again